apps/beeai-ui/next.config.ts

```diff
@@ -79,6 +79,9 @@ const nextConfig: NextConfig = {
     // https://github.com/vercel/next.js/issues/64921
     cssChunking: false,
   },
+  devIndicators: {
+    position: 'bottom-right',
+  },
```

Comment:
Not related to the PR – but ok. Would deserve a separate commit for organization at least.